website/_sass/components/_home.sass

```diff
@@ -0,0 +1,34 @@
+.content
+    height: 100%
+
+.jumbotron
+    height: 750px
+    max-height: 75%
+    min-height: 300px
+    padding-top: 60px
+    box-sizing: border-box
+
+    .visual
+        position: relative
+        width: 100%
+        height: 100%
+        background-image: url(../images/jumbotron.png)
+        background-position: center center
+
```

Comment:
Hey @aciccarello, I have made a build with jekyll and am looking into the website.

It was always a pain to see typedoc "visual" cut on the edge on my phone, like:
![cut](https://user-images.githubusercontent.com/22465806/50539631-c2bb5d80-0bc6-11e9-8a39-eeddb19fb82d.PNG)

If you add 
```suggestion
    background-size: cover;
```
It's going to be 200% prettier!
![notcut](https://user-images.githubusercontent.com/22465806/50539635-d4046a00-0bc6-11e9-97cb-3d751a35a718.PNG)